src/test/kotlin/org/jetBrains/research/tasktracker/tracking/Tasks.kt

```diff
@@ -0,0 +1,52 @@
+@file:Suppress("MatchingDeclarationName")
+
+package org.jetBrains.research.tasktracker.tracking
+
+import org.jetBrains.research.tasktracker.models.Extension
+import org.jetBrains.research.tasktracker.tracking.task.Task
+
+data class MockTask(
+    val filename: String,
+    val ext: Extension,
+    val text: String? = null,
+    val relativePath: String? = null
+) : Task {
+    override fun getContent(): String? {
+        return text
+    }
+
+    override fun getRelativeFilePath(): String? {
+        return relativePath
+    }
+
+    override fun getFileName(): String {
+        return filename
+    }
+
+    override fun getExtension(): Extension {
+        return ext
+    }
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Are you sure about overriding this method? usually in Kotlin you don't need to override equals for data classes

Reply:
I have overridden the equals method in the mockTest class so that we can conveniently retrieve content from VirtualFile and compare the content of tasks from the test data with what we obtained from VirtualFile

Reply:
then don't forget ovveride the hasCode method